Mathlib/Topology/Category/Profinite/Basic.lean

```diff
@@ -145,6 +145,8 @@ def FintypeCat.toProfinite : FintypeCat ⥤ Profinite where
   obj A := Profinite.of A
   map f := ⟨f, by continuity⟩
 
+-- This lemma is generated by `simps` but we don't actually want the `@[simp]` attribute.
+-- https://github.com/leanprover-community/mathlib4/issues/18942
 attribute [nolint simpNF] FintypeCat.toProfinite_map_apply
```

Comment:
Might it be better to do `attribute [-simp] FintypeCat.toProfinite_map_apply` here? That way we can remove the `nolint simpNF`? I'm happy either way round.

Reply:
@b-mehta this doesn't actually work. `attribute [-simp]` is only file local.

Reply:
I've reverted.